src/server/pfs/s3/multipart.go

```diff
@@ -59,11 +59,11 @@ func multipartKeepArgs(path string) (repo string, branch string, key string, upl
 }
 
 func parentDirPath(bucket *Bucket, key string, uploadID string) string {
```

Comment:
Out of scope, but this could be massively improved by using the new filesets API

Reply:
I didn't entirely understand the use case from reading the code, not surprised there's a better way now